### DISCUSSION TITLE
DTI reconstruction - posterior-anterior PA / AP

### DISCUSSION THREAD

**Discussion by dyollb** [dipy/dipy#3289] [Reconstruction] [answered]
I am a complete newbie on the topic of DTI.

So far I managed to use dipy to reconstruct DTI from the IXI dataset.

Now our collaborators have provided DWI data with PA / AP acquisitions. How can I process these?
Should I reconstruct the DTI from each separately and the average the result or is there a better workflow (in dipy)?

**Comment by jhlegarreta** [comment, marked as answer]
AP / PA refer to the polarity of the phase encoding direction of the acquired sequences. AP / PA volumes are acquired to offer a way to correct the distortions that typically appear in EPI sequences, so they are used at the preprocessing stage, prior to fitting your (e.g. DTI) model. So you do not fit your local model into them separately: they are used at the preprocessing stage in order to get single DWI volume where the relevant distortions have been corrected to the extent that is possible.

DIPY by itself does not offer a method to correct such distortions, so you will need to use third-party tools for that purpose. One very well known and extended one is FSL. You can read more about how to use the tools FSL offers towards that end:
https://www.fmrib.ox.ac.uk/primers/intro_primer/ExBox20/IntroBox20.html

There are a few packages out there that perform dMRI data preprocessing steps in a principled way, calling the required FSL methods under the hood and transparently to the user. You can find a comparison here:
https://qsiprep.readthedocs.io/en/latest/comparisons.html

**Reply by jhlegarreta** [reply]
@dyollb did :point_up: answer your question? If yes, please mark it as such so that others can rely on it when visiting the thread. Thanks.
